src/breadcrumbs/examples-list.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+module.exports = {
+  DEFAULT: 'Breadcrumbs with Link component',
```

Comment:
can I override to make vertical aligned breadcrumbs?

Reply:
That should be possible since all styles and the icon can be controlled via `overrides`